docs/pages/access-controls/reference.mdx

```diff
@@ -72,6 +72,17 @@ spec:
     max_sessions: 10
     # permit_x11_forwarding allows users to use X11 forwarding with openssh clients and servers through the proxy
     permit_x11_forwarding: true
+    # Specify whether or not to record the user's desktop sessions.
+    # The cluster's session recording mode must not be set to "off".
```

Comment:
Should we mention `auth_service.session_recording` specifically here?

Reply:
We could, it's either there for static config, or in the `SessionRecordingConfig` for dynamic config.

Reply:
Oh right. Maybe we could add a two-item unordered list after this sentence with the specific config fields?

Reply:
Take a look at the latest commit and LMK what you think.